cwmods/IDA/types.h

```diff
@@ -2,7 +2,11 @@
 #define IDA_CONSTANTS_H
 
 #define _BYTE unsigned char
-#define __int64 long long int
+
+#ifndef __int64

```

Comment:
This does resolve a warning, but I want to redefine anyway, since it's important that any types that IDA spits out are either straight-up primitives or precisely defined by us.